src/hugr/views/sibling.rs

```diff
@@ -0,0 +1,427 @@
+//! Views for HUGR sibling subgraphs.
+//!
+//! Views into subgraphs of HUGRs within a single level of the
+//! hierarchy, i.e. within a sibling graph. Such a subgraph is
+//! given by a root node, of which all nodes in the sibling subgraph are
+//! children, as well as a subgraph boundary, defining the separation between
+//! inside and outside the subgraph. The boundary is fully contained within the
```

Comment:
The *subgraph* is fully contained within the *graph* too, where contained = is-subset-of.

We don't know what a boundary is exactly yet, so the notion of containment of a boundary is necessarily somewhat vague...